openpaas-james/deployment-tests/common/src/main/java/com/linagora/openpaas/deployment/ImapAndSmtpContract.java

```diff
@@ -25,6 +25,8 @@
     @Override
     @BeforeEach
     public void setUp() throws Exception {
+        Thread.sleep(2000);
```

Comment:
At the end I had to put a sleep of 2s in every BeforeEach of tests, as I noticed we seem to have a little latency between the log we are waiting for assuming james is ready, and when the server is really ready. With those, the tests are not unstable anymore.

It's ugly I know...

Reply:
While being acceptable as a (very) short term solution, I would suggest putting an INFO log in `JMXServer.java` and awaiting that log here.

Can you contribute the log addition upstream in apache/james-project , and work asynchronously on removing this thread.sleep?

Reply:
https://github.com/linagora/james-project/pull/4234